sln/WebApiOData.Full.sln

```diff
@@ -0,0 +1,85 @@
+﻿
```

Comment:
If this one is "Full", would that imply it contained the E2E tests as well?

Reply:
WebApiOData.E2E.Full.sln ?